docs/troubleshooting.rst

```diff
@@ -168,3 +168,25 @@ This issue should only happen in development mode. Long story short, it can be s
     tutor dev run lms ./manage.py lms waffle_switch block_structure.invalidate_cache_on_publish on --create
 
 If you'd like to learn more, please take a look at `this Github issue <https://github.com/overhangio/tutor/issues/302>`__.
+
+Reoccuring error "ERR_SOCKET_TIMEOUT" on "tutor images build mfe" 
+-----------------------------------------------------------------
+
+This issue can happen due to a network connection issue if occurs once or twice, but if the issue persists then it is due to high resourse consumption issue related to BuildKit for building simultaneously multiple MFEs enabled.
```

Comment:
Try to make shorter sentences. For instance:

> If this error happens just once or twice, it may be due to a network connection issue. But if the issue persists...

However, it is incorrect to say that a socket timeout error can be caused by high resource consumption. When the build fails because of hardware, the error is different -- can't remember which one, though. See this issue for more information: https://github.com/overhangio/tutor-mfe/issues/125

Please rephrase, either by changing the error message, or by changing the proposed solution.